### DISCUSSION TITLE
docker端启动成功后，发信息收到命令提示失败。

### DISCUSSION THREAD

**Discussion by yaogame** [zhayujie/chatgpt-on-wechat#645] [Q&A]
电脑端用vs运行python一切正常，就是时间久会出错，于是部署了docker端，未设置代理，代理的参数是空的，启动后成功，但是发消息后有下面错误提示，请求大神解答！

Start auto replying.
[INFO][2023-03-28 02:55:49][bridge.py:28] - create bot chatGPT for chat
[INFO][2023-03-28 02:55:50][chat_gpt_bot.py:32] - [OPEN_AI] query=1
[WARNING][2023-03-28 02:55:50][chat_gpt_bot.py:118] - Error communicating with OpenAI: ('Connection aborted.', OSError(22, 'Invalid argument'))
[WARNING][2023-03-28 02:55:50][chat_gpt_bot.py:119] - [OPEN_AI] APIConnection failed
[INFO][2023-03-28 02:55:51][wechat_channel.py:179] - [WX] sendMsg=Reply(type=ERROR, content=ERROR:
我连接不到你的网络), receiver=@7d4ccfd6b33f3c3cd5795ad521874fc4
^CBye~

**Comment by 2734847914** [comment]
一样的问题

**Reply by ACK007** [reply]
求解，一样的问题

**Reply by wstake** [reply]
一样的问题，只不过现在变成“我现在有点累了，等会再来吧”
